README.rst

```diff
@@ -139,24 +139,31 @@ Since we only want to change the behaviour of these community submission request
 
 .. code-block:: python
 
+    from invenio_rdm_records.requests import CommunitySubmission
+    from invenio_rdm_records.services.permissions import RDMRequestsPermissionPolicy
+    from invenio_requests.services.generators import Creator, Receiver
+
+    from invenio_curations.requests.curation import CurationRequest
     from invenio_curations.services.generators import (
         IfCurationRequestAccepted,
         IfRequestTypes,
+        TopicPermission,
     )
-    from invenio_rdm_records.requests import CommunityInclusion, CommunitySubmission
-    from invenio_rdm_records.services.permissions import RDMRequestsPermissionPolicy
-    from invenio_requests.services.generators import Creator, Receiver, Status
+
 
     class CurationRDMRequestsPermissionPolicy(RDMRequestsPermissionPolicy):
         """Customized permission policy for sane handling of curation requests."""
 
-        # Only allow community-submission requests to be accepted after
-        # the rdm-curation request has been accepted
+        curation_request_record_review = IfRequestTypes(
+            [CurationRequest],
+            then_=[TopicPermission(permission_name="can_review")],
```

Comment:
with the new permission generator (see other comment), this basically checks the `record_svc.can_review`?
neat!